cloud/vsphere/cmd/vsphere-machine-controller/main.go

```diff
@@ -34,6 +40,10 @@ var (
 	namedMachinesPath = pflag.String("namedmachines", "", "path to named machines yaml file")
 )
 
+const (
```

Comment:
nit: if you have a single const (or var) you should just put it on one line instead of in a block

`const vsphereMachineControllerName = "..."`